frontend/src/app/modules/interventions/list/list.component.scss

```diff
@@ -1,13 +1,21 @@
 .InterventionsList {
+  padding: 10px;
+  @media (min-width: 700px) {
```

Comment:
Could you please use scss variable `$break-mobile` from `/home/radek/kodawanie/git_repo_lightning/Ekostraz/frontend/src/styles/_breakpoints.scss`?  It has the same exact value as your min-width :)

Reply:
Yes, good point, done 👍